branch.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+    ALTER TABLE participants DROP CONSTRAINT participants_api_key_key;
```

Comment:
It's the uniqueness constraint on `api_key`. We don't need it anymore because the new basic auth relies on user IDs for uniqueness.